### PR TITLE
Add Edit option for notes

### PR DESCRIPTION
**Pull request by glenshelly**

### CODE REVIEW

File: totes [hunk 1]
```diff
@@ -119,6 +119,17 @@ handle_user_input() {
     n | N)
       totes "$@" # propagate arguments
       ;;
+    e | E)
+      if [[ -z "$EDITOR" ]]; then
+        printf "%s\n" "No value for \$EDITOR is set, so unable to edit the requested file."
```
Comment (nathanshelly) [inline totes]: nit: since there's not actually a variable here you could do it all in a single string

File: totes [hunk 2]
```diff
@@ -119,6 +119,17 @@ handle_user_input() {
...
+        exit 0
+      fi
+      "$EDITOR" "$note_to_render"
+      # Date format: [[CC]YY]MMDDhhmm[.ss]
+      # Setting to 1:01 on jan 1, 2001
+      touch -t 0101010101 "$note_to_render"
```
Comment (nathanshelly) [inline totes]: appreciate the `0101010101` timestamp 😄 Might be worth adding a comment mentioning that this puts it at the front of the rotation (could be deduced but I was confused for a sec)

File: totes (hunk 1 above)
Comment (glenshelly) [reply totes]: yes, will do

File: totes (hunk 2 above)
Comment (glenshelly) [reply totes]: will do

**Pull request merged**